extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -307,42 +308,42 @@ private function processCountersForInTrack() {
 
 		$inTrackStatic = AchConfig::getInstance()->getInTrackStatic();
 
-		foreach ( $this->mCounters as $badge_type_id => $badge_counter ) {
+		foreach( $this->mCounters as $badge_type_id => $badge_counter ) {
 
 			$badgeType = AchConfig::getInstance()->getBadgeType( $badge_type_id );
 
-			if ( $badgeType == BADGE_TYPE_INTRACKSTATIC || $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY ) {
+			if( $badgeType == BADGE_TYPE_INTRACKSTATIC || $badgeType == BADGE_TYPE_INTRACKEDITPLUSCATEGORY ) {
 
-				if ( $badge_type_id == BADGE_LOVE ) {
-					$eventsCounter = $badge_counter[COUNTERS_COUNTER];
-				} else if ( $badge_type_id == BADGE_BLOGCOMMENT ) {
+				if( $badge_type_id == BADGE_LOVE ) {
+					$eventsCounter = $badge_counter[ COUNTERS_COUNTER ];
+				} else if( $badge_type_id == BADGE_BLOGCOMMENT ) {
 					$eventsCounter = count( $badge_counter );
-				} else if ( $badge_type_id == BADGE_SHARING ) {
+				} else if( $badge_type_id == BADGE_SHARING ) {
 					$eventsCounter = -1;
-					if ( isset( $badge_counter['article_ids'] ) ) {
+					if( isset( $badge_counter[ 'article_ids' ] ) ) {
 						$eventsCounter = 0;
-						foreach ( $badge_counter['article_ids'] as $article_id => $ips ) {
+						foreach( $badge_counter[ 'article_ids' ] as $article_id => $ips ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$article_id" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)